bnelson777/id.ly
# IL-103 - Indicate read messages in inbox

Testing instructions:
[Dev] Clear All Data
Reload app
Inbox
Check that inbox contains two messages:
  First is read(grayed out)
  Second is unread(not grayed out)

Note: Adding messages will create read messages(added ticket in jira).
